### DISCUSSION TITLE
I would like to volunteer for localize MDN content for 'Bangla' Language

### DISCUSSION THREAD

**Discussion by impactnazmul** [mdn/mdn-community#304] [Localization] [answered]
I have been learning web development and most of my reference checking is done with MDN. So, I would like to translate the contents of MDN's knowledge pages for my community so that they can learn web development in their language.

**Comment by schalkneethling** [comment, marked as answer]
Hi there @mn-huda,

Thank you for your interest and offer to translate the contents of MDN Web Docs in Bangla. There is a process in place for how to go about this which you can find here.

- General information about localization - https://developer.mozilla.org/en-US/docs/MDN/Community/Contributing/Translated_content
- Reactivating or activating a new locale - https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale
- MDN localization update blog post - https://hacks.mozilla.org/2021/02/mdn-localization-update-february-2021/

I hope these resources will answer your questions.

**Comment by pransh15** [comment]
Closing this since it was answered.
